theme/static/style/style.css

```diff
@@ -98,6 +98,113 @@ ul.topnav li [name="q"] {
     border: 0;
 }
 
+.topnav{
+    padding: 10px;
+    overflow: hidden;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0, .3);
+    z-index: 1;
+}
+
+.topnav ul{
+    list-style-type: none;
+    text-align: right;
+}
+
+.topnav li{
+    display: inline-block;
+    margin-left: 15px;
+    vertical-align: top;
+}
+
+.topnav ul a{
+    display: inline-block;
+    padding: 0 16px;
+    text-align: center;
+    line-height: 50px;
+    border-radius: 3px;
+}
+
+.topnav .searchbox{
+    display: inline-block;
+    height: 50px;
+    padding-left: 25px;
+    border-radius: 3px;
+    border: 0;
+    box-sizing: border-box;
+    font-size: 1.6rem;
+}
+
+.topnav .open-menu{
+    display: none;
+    width: 50px;
+    height: 50px;
+    border-radius: 3px;
+    border: 0;
+    cursor: pointer;
+}
+
+.mobilenav{
+    position: fixed;
+    top: -100%;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 2;
+    transition: .5s top ease;
+    overflow-y: scroll;
+}
+
+.mobilenav  .close-menu{
+    position: absolute;
+    top: 10px;
+    left: 10px;
```

Comment:
Replace the `px` units with `rem` or any other units that are commonly used in this stylesheet.